website/screens/components/badge/specs/BadgeSpecsPage.tsx

```diff
@@ -33,7 +37,7 @@ const sections = [
     title: "Design tokens",
     content: (
       <DxcParagraph>
-        This component does not currently have design tokens.
+        This component currently has no design tokens.
       </DxcParagraph>
```

Comment:
If we are going to use this message we can replace it also in the divider and status light specs. How do you see it? @GomezIvann

Reply:
I was trying to standardise it because it was different in each place but I forgot those! Thanks Raquel I change them.